fronts-client/src/components/FrontsEdit/CollectionComponents/Sublinks.tsx

```diff
@@ -98,8 +100,7 @@ class Sublinks extends React.Component<SublinkProps> {
     );
   }
 
-  private dragEventNotDenylisted = (e: React.DragEvent) =>
-    !dragEventIsDenylisted(e, collectionDropTypeDenylist);
+  private dragEventNotDenylisted = (e: React.DragEvent) => !denyDragEvent()(e);
```

Comment:
Not sure I understand what this is for - is in why the double-negative, and whether it would be clearer to remove the indirection through the method given it's just a single call?

Reply:
Yes, I see no reason not to move to an inline method! The reflex to use a method may have been related to a common pattern when class components were used and arrow fns were useful for lexical scoping, but that doesn't apply here. 88ec333aa9483a59f59cb445b23d5e0e050e70d0